tests/application/controllers/signup.spec.ts

```diff
@@ -43,12 +44,30 @@ const makePasswordValidator = (): PasswordValidator => {
 const makeAddGuardian = (): AddGuardian => {
   class AddGuardianStub implements AddGuardian {
     async add (guardian: AddGuardian.Params): Promise<AddGuardian.Result> {
-      return await new Promise(resolve => { resolve(true) })
+      return {
+        id: 1,
+        firstName: 'any_first_name',
+        lastName: 'any_last_name',
+        email: 'any_email@mail.com',
+        phone: 'any_phone'
+      }
     }
   }
   return new AddGuardianStub()
 }
 
+const makeFakeRequest = (): HttpRequest => ({
+  body: {
+    firstName: 'any_first_name',
+    lastName: 'any_last_name',
+    email: 'any_email@mail.com',
+    phone: 'any_phone',
+    password: 'any_password',
+    passwordConfirmation: 'any_password',
+    isPrivacyPolicyAccepted: true
```

Comment:
isPrivacyPolicyAccepted has been removed from the database and should be fixed in testing

Reply:
I think there is no problem keeping this field in request, as it makes sense not to accept the registration of a user who does not accept the privacy policy, it would be a layer of security in addition to front validation. Make sense?